contracts/contracts/rem.system/src/producer_pay.cpp

```diff
@@ -125,7 +136,9 @@ namespace eosiosystem {
             _producers.modify(prod, same_payer, [&](auto& p) {
```

Comment:
what if we move this `modify` to the previous `for` and then replace this `for` with the call to `update_pervote_shares()`?

Reply:
Done